### PR TITLE
Added functions

### PR DESCRIPTION
**Pull request by MarcoLPR**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "extends": [
+      "tslint:recommended",
+      "tslint-react"
```
Comment (geoperez) [inline tslint.json]: Remove TsLint React

File: src/getAverage.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export const getAverage = (data: any[], fixed?: number, field?: string) => {
```
Comment (geoperez) [inline src/getAverage.ts]: 👎

File: src/getMedian.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { sortValueByAsc } from './sortValueByAsc';
```
Comment (geoperez) [inline src/getMedian.ts]: 👎

File: src/groupedCount.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export const groupedCount = (data: any[], prop: string) => data
```
Comment (geoperez) [inline src/groupedCount.ts]: 👎

File: src/humanize.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export function humanize(name: string) {
```
Comment (geoperez) [inline src/humanize.ts]: 👍

File: src/numberWithCommas.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+export const numberWithCommas = (int: number) => int.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```
Comment (geoperez) [inline src/numberWithCommas.ts]: 👎

File: src/objectDifference.ts [hunk 7]
```diff
@@ -0,0 +1,21 @@
+export const objectDifference = (obj1: any, obj2: any) => {
```
Comment (geoperez) [inline src/objectDifference.ts]: 👍

File: src/removeDuplicated.ts [hunk 8]
```diff
@@ -0,0 +1,4 @@
+export const removeDuplicates = (array: any[], prop: string) => {
```
Comment (geoperez) [inline src/removeDuplicated.ts]: 👍

File: src/sortValueByAsc.ts [hunk 9]
```diff
@@ -0,0 +1,9 @@
+export const sortValueByAsc = (data: any[], prop?: string, isDate?: boolean) => data.sort((a: any, b: any) => {
```
Comment (geoperez) [inline src/sortValueByAsc.ts]: 👎

File: src/sortValueByDesc.ts [hunk 10]
```diff
@@ -0,0 +1,9 @@
+export const sortValueByDesc = (data: any[], prop?: string, isDate?: boolean) => data.sort((a: any, b: any) => {
```
Comment (geoperez) [inline src/sortValueByDesc.ts]: 👎

File: src/toTitleCase.ts [hunk 11]
```diff
@@ -0,0 +1,8 @@
+export function toTitleCase(title: string) {
```
Comment (geoperez) [inline src/toTitleCase.ts]: 👍

File: src/withEnter.ts [hunk 12]
```diff
@@ -0,0 +1,11 @@
+export const onEnterKey = (callback: () => void) => (ev: { keyCode: number; }) => {
```
Comment (geoperez) [inline src/withEnter.ts]: 👍

File: tslint.json [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+        true,
+        "check-branch",
+        "check-decl",
+        "check-operator",
+        "check-separator",
+        "check-type"
+      ]
+    },
+    "rules": {
+      "jsx-no-multiline-js": false,
```
Comment (geoperez) [inline tslint.json]: Remove jsx rules

File: src/fetchController.ts [hunk 14]
```diff
@@ -0,0 +1,75 @@
+function defaultHeadersResolver(url: string, accessToken: string) {
```
Comment (geoperez) [inline src/fetchController.ts]: 👍

File: src/debounce.ts [hunk 15]
```diff
@@ -0,0 +1,24 @@
+// based on https://github.com/chodorowicz/ts-debounce/blob/master/src/index.ts
```
Comment (geoperez) [inline src/debounce.ts]: 👍

File: src/colorGenerator.ts [hunk 16]
```diff
@@ -0,0 +1,19 @@
+const interpolation = (a: number[], b: number[], factor: number) => {
```
Comment (geoperez) [inline src/colorGenerator.ts]: 👍

File: package.json [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+  "bugs": {
+    "url": "https://github.com/unosquare/uno-js/issues"
+  },
+  "homepage": "https://github.com/unosquare/uno-js#readme",
+  "devDependencies": {
+    "@types/jasmine": "^3.3.15",
+    "jasmine": "^3.1.0",
+    "jasmine-ts": "^0.2.1",
+    "ts-node": "^8.5.4",
+    "tslint": "^5.20.1",
```
Comment (geoperez) [inline package.json]: Change to Eslitn, ask @kadosh

File: src/getAverage.ts (hunk 2 above)
Comment (kadosh) [inline src/getAverage.ts]: 👎

File: src/getMedian.ts [hunk 18]
```diff
@@ -0,0 +1,8 @@
...
+
+export const getMedian = (data: any[], field?: string) => {
```
Comment (kadosh) [inline src/getMedian.ts]: 👎

File: src/groupedCount.ts (hunk 4 above)
Comment (kadosh) [inline src/groupedCount.ts]: 👎

File: src/sortValueByAsc.ts (hunk 9 above)
Comment (kadosh) [inline src/sortValueByAsc.ts]: 👎

File: src/sortValueByDesc.ts (hunk 10 above)
Comment (kadosh) [inline src/sortValueByDesc.ts]: 👎

**Pull request merged**